### ISSUE TITLE
[Mangafast] Connector not working

### ISSUE THREAD

**Issue by MikeZeDev** [manga-download/hakuneko#3756]
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
No. They seems to have borked their update i cant list mangas, even by genre. Tried on FF, FF nightly, and even HN internal browser. But maybe its just me.

**Describe the bug**
* I cant update manga list because HN get a 404 error for https://mangafast.net/list-manga/page/1 . 
* Tried to paste the first manga on their webpage : https://mangafast.net/read/dont-flirt-me-lord-angel doesnt work

Since i have no MF bookmark and i cant add one i cant test more to see if we are able to get chapters and files :/
On my browser i am able to see and read manga from the front page properly but thats all.

**To Reproduce**
Trying to update manga list. Doesnt work
Pasting https://mangafast.net/read/dont-flirt-me-lord-angel doesnt recognize the manga.

**Screenshots**
no need.

**HakuNeko (please complete the following information):**
 - Nightly

**Comment by MikeZeDev** [comment]
Check https://mangafast.net/list-manga there is a JS error in extends.JS (obfuscation fail ? )  Nothing HN can do. At least now we have an issue open to remind us about that problem. Sorry for disturbing.

**Comment by Sheepux** [comment]
They simply changed their website:
Also https://mangafast.net/list-manga/2 to https://mangafast.net/list-manga/10 are using different template than the other pages (see https://mangafast.net/list-manga/11 ).
I tested and it looks like we can safely ignore those pages (tested a few titles, and they were present in the other pages).

I also had to introduce a small 100ms throttling or i would get a lot of http 50x when downloading the pages.
PR in review within the week.
